docs-source/content/userguide/managing-fmw-domains/soa-suite/WebLogic-Logging-Exporter-Setup.md

```diff
@@ -8,17 +13,15 @@ This documents provides the steps to publish WebLogic Server logs into Elasticse
 
 ## Prerequisite
 
-This document assumes that you have already setup Elasticsearch/Kibana for logs collection. If you have not, please refer this [document](/kubernetes/samples/scripts/elasticsearch-and-kibana/README.md) to setup Elasticsearch/Kibana from WebLogic operator samples.
+This document assumes that you have already set up Elasticsearch and Kibana for logs collection. If you have not, please refer this [document](/kubernetes/samples/scripts/elasticsearch-and-kibana/README.md) to set up Elasticsearch and Kibana from WebLogic operator samples.
```

Comment:
WebLogic operator -> In the document, you can use the term "operator" to refer to the WebLogic Kubernetes Operator (its official name).